android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -70,8 +70,8 @@ class GetUserMediaImpl {
         }
 
         if (camera2supported) {
-            Log.d(TAG, "Creating video capturer using Camera2 API.");
-            cameraEnumerator = new Camera2Enumerator(reactContext);
+            Log.d(TAG, "Creating video capturer using Camera2 API with UVC support.");
+            cameraEnumerator = new UVCCamera2Enumerator(reactContext);
```

Comment:
Not sure if there's any issues if for some reason a device doesn't support USB hosting, but to be safe, it might good to add a check for `PackageManager.hasSystemFeature(PackageManager.FEATURE_USB_HOST)` and fall back to the regular Camera2Enumerator if it can't support USB cameras.

Reply:
Good call.

Reply:
Very good call, starting to think i might not have any devices with that feature 🤔

Reply:
This should not be an issue since the implementation extends the Camera2Enumerator class (and uvc device enumeration exceptions are handled). But i will add the FEATURE_USB_HOST check just in case.

Reply:
Doesn't hurt sometimes to try and be a bit more safe.
Can be the difference between pushing forward and dropping off a cliff 😄